GitCommands/Git/Commands/GitCommandHelpers.cs

```diff
@@ -69,10 +69,10 @@ public static ArgumentString AddSubmoduleCmd(string remotePath, string localPath
                         ? (ArgumentString)"--no-optional-locks"
                         : default)
                 {
-                    "--no-color",
-                    { staged, "-M -C --cached" },
-                    extraDiffArguments,
+                    { "-M -C" },
```

Comment:
For me, the shortcut options (with one '-') should be kept for humans in a terminal.

While you are at it, is it possible to replace it by the full option name so that reading code and also debugging could be made easier? 

I had to look at the documentation for these obscure options and it seems to be the shortcuts for '--find-renames' and '--find-copies' which is much more comprehensive, right?

Reply:
will change

Reply:
done